code/modules/mob/living/living_update_icons.dm

```diff
@@ -20,7 +20,7 @@
 		current_size *= resize
 		//Update the height of the maptext according to the size of the mob so they don't overlap.
 		var/old_maptext_offset = body_maptext_height_offset
-		body_maptext_height_offset = initial(maptext_height) * (current_size - 1)
+		body_maptext_height_offset = initial(maptext_height) * (current_size - 1) * 0.5 //SKYRATE EDIT: Until it's fixed upstream
```

Comment:
I take it it's still not fixed upstream?

Reply:
Yup, A PR is up but I would rather not delay this another time just because of one line.